src/applications/pensions/components/IntroductionPage.jsx

```diff
@@ -9,10 +9,6 @@ class IntroductionPage extends React.Component {
     focusElement('.va-nav-breadcrumbs-list');
   }
 
-  goForward = () => {
```

Comment:
Why was this removed?

Reply:
goFoward wasn't being called anywhere in the component